src/openvic-simulation/economy/production/Employee.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "openvic-simulation/pop/Pop.hpp"
+
+namespace OpenVic {
+	struct Employee {
+	private:
+		Pop::pop_size_t PROPERTY_RW(size);
+	public:
+		Pop& pop;
+		Employee(Pop& new_pop, Pop::pop_size_t const new_size);
```

Comment:
If you want a `const` for `new_size` then put it before the type rather than after (same in the C++ file), although `const` qualifiers for non-pointer or reference function arguments in declarations don't do anything, you could remove the `const` from the header or from the cpp file and the other one would still work.